snappy/firstboot.go

```diff
@@ -46,39 +47,63 @@ func wrapConfig(pkgName string, conf interface{}) ([]byte, error) {
 	return yaml.Marshal(configWrap)
 }
 
-var activeSnapByName = ActiveSnapByName
-var activeSnapsByType = ActiveSnapsByType
+var newPkgmap = newPkgmapImpl
+
+func newPkgmapImpl() (map[string]Part, error) {
```

Comment:
Is it a part, a package, or a snap?  *mind blown*

Unrelated to this branch, though. We'll fix it some day.